src/components/sdk/DeviceSelection/CameraSelection/VideoFilterSelection.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { ChangeEvent, useEffect, useState } from 'react';
+
+import { FormField } from '../../../ui/FormField';
+import { Select } from '../../../ui/Select';
+import { useBackgroundBlur } from '../../../../providers/BackgroundBlurProvider';
+import { useMeetingManager } from '../../../../providers/MeetingProvider';
+import { isVideoTransformDevice } from 'amazon-chime-sdk-js';
+
+interface Props {
+  /** Label shown for video filter selection, by default it is "Video Filter" */
+  label?: string;
+  /** Label shown in the dropdown when no video filter has been selected yet, by default it is "Select video filter" */
+  labelForUnselected?: string;
+}
+
+export const VideoFilterSelection: React.FC<Props> = ({
```

Comment:
Should we not include this component as part of the library but a component in the demo? Selection can be a array or a checkbox based on UX since we have a single processor so I think meeting demo component as an example would be more appropriate and based on customer requests we can pull this as a component in library. Just a thought.

Reply:
There are other selection components in the library - but this component is not "customizable" all that much. I think thats why we might want it as a demo component, but anyone who uses the BackgroundBlurProvider might want this component, and we wouldnt be releasing this component if it is only in the demo component. So I think we can include it in the react lib.